src/App.tsx

```diff
@@ -17,18 +61,29 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setStatus={setStatus}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todoList.length ? (
```

Comment:
If you get an empty array from the API, a user will see an infinite loader. You should show loader only when you actually load something